docs/supported_apis/dataframe_supported.rst

```diff
@@ -133,26 +145,30 @@ default to pandas.
 +----------------------------+---------------------------+------------------------+----------------------------------------------------+
 | ``fillna``                 | `fillna`_                 | P                      | ``value`` parameter of type DataFrame defaults to  |
 |                            |                           |                        | pandas                                             |
+|                            |                           |                        | **Omnisci**: ``P``                                 |
```

Comment:
Why there is no any note about what parameters are unsupported as specified in other places in the case of `P` for OmniSci?

Reply:
Right, "P" should be elaborated.

You can find a list of unsupported parameters here:
https://github.com/modin-project/modin/blob/95d71b2bde0ceb40cb1b64fa762b3009a94168dd/modin/experimental/core/execution/native/implementations/omnisci_on_native/dataframe/dataframe.py#L619-L629

It also supports only dictionary and scalar as a filling `value` [[1]](https://github.com/modin-project/modin/blob/95d71b2bde0ceb40cb1b64fa762b3009a94168dd/modin/experimental/core/execution/native/implementations/omnisci_on_native/dataframe/dataframe.py#L633-L653)